util/local_storage_utils.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020-2021 the Deno authors. All rights reserved. MIT license.
+
+export function getItem(key: string) {
+  if (typeof window === "undefined") {
+    return null;
+  }
```

Comment:
```suggestion
```

I guess this is not required as the below try block will throw when accessing a property of undefined?

Reply:
makes sense